Example/SmileID/BiometricKYC/BiometricKycWithIdInputScreenViewModel.swift

```diff
@@ -10,8 +10,8 @@ enum BiometricKycWithIdInputScreenStep {
 }
 
 class BiometricKycWithIdInputScreenViewModel: ObservableObject {
-    private let userId = generateUserId()
-    private let jobId = generateJobId()
+    let userId = generateUserId()
```

Comment:
Could we do this in the `biometricKycScreen` screen instead of the viewmodel?

Reply:
The change that was made there was to remove `private` from the properties (if you check what we have on main branch). Also the `userId` and `jobId` are needed here for the auth requests in the view model or Is there a particular reason why you think it should be done in the view?

Reply:
The reason I asked is because on `SmileID.swift`, we do something like this


```swift
public class func biometricKycScreen(
        idInfo: IdInfo,
        userId: String = generateUserId(),
        jobId: String = generateJobId(),
```

We could make the implementation similar on the example, so that the SwiftUI component passes the `userId` and `jobId` inside the viewmodel (I think this is neater) instead of referencing a userId generated inside the viewmodel and passing it again like this file 

`Example/SmileID/BiometricKYC/BiometricKycWithIdInputScreen.swift`

```swift
case .sdk(let idInfo):
            SmileID.biometricKycScreen(
                idInfo: idInfo,
                userId: viewModel.userId,
                jobId: viewModel.jobId,
                allowAgentMode: true,
                delegate: delegate
            )
```

So it would look like this

```swift
case .sdk(let idInfo):
            SmileID.biometricKycScreen(
                idInfo: idInfo,
                userId: generateUserId(),
                jobId: generateJobId(),
                allowAgentMode: true,
                delegate: delegate
            )
```

not sure if that makes sense

Reply:
We need to save the userId somewhere, so that we know the value when persisting the job. Also iirc, we used to have it like this but ran into an issue where the view kept generating new user id's on every render which led to issues for things like authentication (cc @JNdhlovu)